### PR TITLE
@trezor/transport follow-up

### PR DESCRIPTION
**Pull request by mroz22**
- Long.js was not loaded properly in browser environement. Reported also in protobufjs repo https://github.com/protobufjs/protobuf.js/issues/758
- Added transport package to CI 
- Fixed packages in CI not deploying because there was no build running

### CODE REVIEW

File: packages/transport/tests/encode-decode-basic.test.js
Comment (prusnak) [inline packages/transport/tests/encode-decode-basic.test.js:135]: Can we have also similar fixture but for big negative Sint64?

File: ci/deploy.yml
Comment (vdovhanych) [inline ci/deploy.yml]: Assume that `--dry-run` for testing only and we will change it once ready for the merge right?

Comment (mroz22) [reply ci/deploy.yml]: ah yes. forgot about that

**Review by matejkriz — APPROVED** [review]

**Pull request merged**
